uhyve-interface/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "uhyve-interface"
+version = "0.1.0"
+edition = "2021"
+authors = [
+  "Jonathan Klimt <jonathan.klimt@eonerc.rwth-aachen.de>",
+  "Stefan Lankes <slankes@eonerc.rwth-aachen.de>",
+]
+description = "The interface between uhyve and a guest VM"
+repository = "https://github.com/hermitcore/uhyve"
+license = "MIT OR Apache-2.0"
+keywords = ["hypervisor", "unikernel", "hermit"]
+categories = ["os"]
+
+[dependencies]
+num_enum = { version = "0.5.7", default-features = false }
```

Comment:
Just a nit, but unless we depend on features introduced in patch version 7, it would create less noise in this file to omit the patch version altogether.

Same for the next dependency.